pilot/pkg/security/authz/builder/testdata/http/allow-full-rule-out.yaml

```diff
@@ -240,38 +240,46 @@ typedConfig:
             ids:
             - orIds:
                 ids:
-                - authenticated:
-                    principalName:
+                - filterState:
+                    key: io.istio.peer_principal
```

Comment:
Any chance to have istio-independent key names for the common TLS properties ? I don't mind if it's a short term thing.

Reply:
It's an internal name, but can be changed. It's recommended to be reverse DNS, what's the suggestion?

Reply:
Can  we use a name with envoy ? I would be best if this we not viewed as istio specific, and other gateway impl using envoy would use same mechanism.